configure.ac

```diff
@@ -766,6 +766,12 @@ AC_C_BIGENDIAN
 AC_CHECK_HEADERS([endian.h])
 AC_CHECK_DECLS(le32toh, [], [], [[#include <endian.h>]])
 
+AC_CHECK_DECLS(closefrom, [], [], [[#include <unistd.h>]])
+AC_REPLACE_FUNCS(closefrom)
```

Comment:
BSD and Solaris. AIX equivalent is the ``F_CLOSEM`` call. Linux has no equivalent.